.github/workflows/documentation-windows.yml

```diff
@@ -2,7 +2,7 @@ name: Documentation Windows
 
 on:
   push:
-    branches: [ master, develop, debug_windows_doc ]
+    branches: [ master, develop, 9818_Windows_documentation ]
```

Comment:
Force build this branch, or we won't know until its merged to develop. Running here: https://github.com/NREL/EnergyPlus/actions/runs/4015981786